src/test/java/DemoqaTests.java

```diff
@@ -0,0 +1,119 @@
+        Assertions.assertEquals("Registration Form", modalTitle);
+        driver.findElement(By.id("firstName")).sendKeys("Bob");
+        driver.findElement(By.id("lastName")).sendKeys("Marley");
+        driver.findElement(By.id("userEmail")).sendKeys("test@gmail.com");
+        driver.findElement(By.id("age")).sendKeys("20");
+        driver.findElement(By.id("salary")).sendKeys("3500");
+        driver.findElement(By.id("department")).sendKeys("musician");
+        driver.findElement(By.id("submit")).click();
+        driver.findElement(By.id(SEARCH_BOX)).sendKeys("Bob");
+        var result = driver.findElement(By.ByCssSelector.cssSelector(FIRST_CHILD)).getText();
+        Assertions.assertEquals("Bob", result);
+    }
+
+    @Test
+    @DisplayName("Check the double click button")
+    void doubleClickClickTest() {
+        driver.get(BASIC_URL + "buttons");
+        var doubleClickElement = driver.findElement(By.id(DOUBLE_CLICK_BTN));
+        new Actions(driver)
+                .doubleClick(doubleClickElement)
+                .perform();
+        var message = driver.findElement(By.id(DOUBLE_CLICK_MESSAGE)).getText();
+        Assertions.assertEquals("You have done a double click", message);
+    }
+
+    @Test
+    @DisplayName("Check the right click button")
+    void rightClickClickTest() {
+        driver.get(BASIC_URL + "buttons");
+        var doubleClickElement = driver.findElement(By.id(RIGHT_CLICK_BTN));
+        new Actions(driver)
+                .contextClick(doubleClickElement)
+                .perform();
+        var message = driver.findElement(By.id(RIGHT_CLICK_MESSAGE)).getText();
+        Assertions.assertEquals("You have done a right click", message);
+    }
+
+    @Test
+    @DisplayName("Check the registration form")
+    void registrationTest() {
+        driver.get(BASIC_URL + "register");
+        driver.findElement(By.id("firstname")).sendKeys("Valeriy");
+        driver.findElement(By.id("lastname")).sendKeys("Zhmyshenko");
+        driver.findElement(By.id("userName")).sendKeys("Zhmyleriy");
+        driver.findElement(By.id("password")).sendKeys("Qwerty123$");
+        var iframe = driver.findElement(cssSelector(IFRAME_TITLE_RE_CAPTCHA));
+        driver.switchTo().frame(iframe);
+        driver.findElement(By.id(RECAPTCHA_ANCHOR)).click();
+        driver.switchTo().parentFrame();
+        driver.findElement(By.id(REGISTER_BUTTON)).click();
+        var successMessage = driver.switchTo().alert().getText();
+        Assertions.assertEquals("User Register Successfully.", successMessage);
+    }
+
+    @Test
+    @DisplayName("Check the login form")
+    void loginTest() {
+        driver.get(BASIC_URL + "login");
+        driver.findElement(By.id("userName")).sendKeys("Zhmyleriy");
+        driver.findElement(By.id("password")).sendKeys("Qwerty123$");
+        driver.findElement(cssSelector(BUTTON_LOGIN)).click();
+        WebElement userNameElement = waitForElementVisibility(By.id(USER_NAME_VALUE));
```

Comment:
do not mess reference type `WebElement` with `var` in the next line. Stick to one code style